_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -0,0 +1,383 @@
+			$count_data = Akismet_Admin::get_stats( Akismet::get_api_key() );
+		} else {
+			return $status->get_error_messages();
+		}
+
+		if ( 'all' === $data['date'] ) {
+			return $count_data['all']->spam;
+		}
+
+		// Organize the requested date time to YYYY-MM
+		$data['date'] = DateTime::createFromFormat( 'Ym', $data['date'] );
+		return $count_data['6-months']->breakdown->{ $data['date']->format( 'Y-m' ) }->spam;
+	}
+
+	/**
+	 * Get date of last downtime.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return mixed|WP_Error Number of days since last downtime. Otherwise, a WP_Error instance with the corresponding error.
+	 */
+	public static function monitor_get_last_downtime() {
+		if ( Jetpack::is_module_active( 'monitor' ) ) {
+			$monitor       = new Jetpack_Monitor();
+			$last_downtime = $monitor->monitor_get_last_downtime();
+			if ( is_wp_error( $last_downtime ) ) {
+				return $last_downtime;
+			} else {
+				return rest_ensure_response( array(
+					'code' => 'success',
+					'date' => human_time_diff( strtotime( $last_downtime ), strtotime( 'now' ) ),
+				) );
+			}
+		}
+
+		return new WP_Error( 'not-active', esc_html__( 'The requested Jetpack module is not active.', 'jetpack' ), array( 'status' => 404 ) );
+	}
+
+	/**
+	 * Get number of plugin updates available.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return mixed|WP_Error Number of plugin updates available. Otherwise, a WP_Error instance with the corresponding error.
+	 */
+	public static function get_plugin_update_count() {
+		$updates = wp_get_update_data();
+		if ( isset( $updates['counts'] ) && isset( $updates['counts']['plugins'] ) ) {
+			$count = $updates['counts']['plugins'];
+			if ( 0 == $count ) {
+				$response = array(
+					'code'    => 'success',
+					'message' => esc_html__( 'All plugins are up-to-date. Keep up the good work!', 'jetpack' ),
+					'count'   => 0,
+				);
+			} else {
+				$response = array(
+					'code'    => 'updates-available',
+					'message' => esc_html( sprintf( _n( '%s plugin need updating.', '%s plugins need updating.', $count, 'jetpack' ), $count ) ),
+					'count'   => $count,
+				);
+			}
+			return rest_ensure_response( $response );
+		}
+
+		return new WP_Error( 'not-found', esc_html__( 'Could not check updates for plugins on this site.', 'jetpack' ), array( 'status' => 404 ) );
+	}
+
+	/**
+	 * Get services that this site is verified with.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return mixed|WP_Error List of services that verified this site. Otherwise, a WP_Error instance with the corresponding error.
+	 */
+	public static function get_verified_services() {
+		if ( Jetpack::is_module_active( 'verification-tools' ) ) {
+			$verification_services_codes = get_option( 'verification_services_codes' );
+			if ( is_array( $verification_services_codes ) && ! empty( $verification_services_codes ) ) {
+				$services = array();
+				foreach ( jetpack_verification_services() as $name => $service ) {
```

Comment:
I get a fatal when calling this because `jetpack_verification_services()` isn't defined.  We'll need to require `/modules/verification-tools/blog-verification-tools.php` somewhere before this.